tools/docker/runner/entrypoint.sh

```diff
@@ -8,10 +8,15 @@ run() {
 run ip link add br-lan type bridge
 run ip link add wlan0 type dummy up
 run ip link add wlan2 type dummy up
+run ip link add prplmesh-eth0 type dummy up
+run ip link set prplmesh-eth0 master br-lan
 run ip link set eth0 master br-lan
 run ip link set wlan0 master br-lan
 run ip link set wlan2 master br-lan
 run ifconfig eth0 0.0.0.0
+run ifconfig prplmesh-eth0 up
+run ifconfig wlan0 up
+run ifconfig wlan2 up
```

Comment:
Why are these three suddenly needed? Isn't the `ip link up` that is done a bit higher sufficient?

Reply:
no, ip link add doesn't actually support adding an interface and bringing it up in a single command.

Reply:
but I will change it all to use ip command.